web-client/src/app/state/session-xrpl.service.ts

```diff
@@ -21,13 +22,13 @@ import { environment } from 'src/environments/environment';
 import { ifDefined } from 'src/helpers/helpers';
 import { TransactionSigned, TransactionToSign } from 'src/schema/actions';
 import * as xrpl from 'xrpl';
+import { TxResponse } from 'xrpl';
```

Comment:
to keep the imports consistent, can you remove this line and use `xrpl.TxResponse` where you need it?